### PR TITLE
#1532 With multiplatform implementation, storing the subscriptions was modifie…

### PR DESCRIPTION
**Pull request by shwethanidd**
…d which breaks when we run it with a platform using previous version of message bus implementation. Added a fix to store the subscription in default 'internal' subscription entry.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/VOLTTRON/volttron/pull/1533%23discussion_r145235467%22%2C%20%22https%3A//github.com/VOLTTRON/volttron/pull/1533%23issuecomment-337349732%22%2C%20%22https%3A//github.com/VOLTTRON/volttron/pull/1533%23issuecomment-337392900%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/VOLTTRON/volttron/pull/1533%23issuecomment-337349732%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40shwethanidd%20can%20you%20make%20the%20releases/5.0rc%20branch%20the%20base%20instead%20of%20develop.%20%20I%20will%20merge%20it%20back%20this%20week%20into%20develop%22%2C%20%22created_at%22%3A%20%222017-10-17T19%3A50%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/3979063%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/craig8%22%7D%7D%2C%20%7B%22body%22%3A%20%22Changing%20base%20to%205.0rc%22%2C%20%22created_at%22%3A%20%222017-10-17T22%3A25%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/19616893%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/shwethanidd%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2070c5cb2884719c22121afa164f02043313ab85e0%20volttron/platform/vip/pubsubservice.py%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/VOLTTRON/volttron/pull/1533%23discussion_r145235467%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Rather%20than%20this%20use%20msg.get%28%27all_platforms%27%2C%20False%29%22%2C%20%22created_at%22%3A%20%222017-10-17T19%3A46%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/3979063%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/craig8%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20volttron/platform/vip/pubsubservice.py%3AL212-222%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 70c5cb2884719c22121afa164f02043313ab85e0 volttron/platform/vip/pubsubservice.py 5'></a> &nbsp;<b><a href='https://github.com/VOLTTRON/volttron/pull/1533#discussion_r145235467'>File: volttron/platform/vip/pubsubservice.py:L212-222</a></b>
- <a href='https://github.com/craig8'></a> Rather than this use msg.get('all_platforms', False)
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/VOLTTRON/volttron/pull/1533#issuecomment-337349732'>General Comment</a></b>
- <a href='https://github.com/craig8'></a> @shwethanidd can you make the releases/5.0rc branch the base instead of develop.  I will merge it back this week into develop
- <a href='https://github.com/shwethanidd'></a> Changing base to 5.0rc

<a href='https://www.codereviewhub.com/VOLTTRON/volttron/pull/1533?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/VOLTTRON/volttron/pull/1533?approve=1'></a>&nbsp;<a href='https://github.com/VOLTTRON/volttron/pull/1533'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by craig8** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/VOLTTRON/volttron/pull/1533'>Refresh</a>.<a href='#crh-update-none'></a>

File: volttron/platform/vip/pubsubservice.py [hunk 1]
```diff
@@ -212,7 +212,10 @@ def _peer_subscribe(self, frames):
             peer = frames[0].bytes
             prefix = msg['prefix']
             bus = msg['bus']
-            is_all = msg['all_platforms']
+            try:
```
Comment (craig8) [inline volttron/platform/vip/pubsubservice.py]: Rather than this use msg.get('all_platforms', False)

**Comment by craig8** [comment]
@shwethanidd can you make the releases/5.0rc branch the base instead of develop.  I will merge it back this week into develop

**Comment by shwethanidd** [comment]
Changing base to 5.0rc

**Comment by craig8** [comment]
Was done with #1534

**Pull request closed without merging**